### PR TITLE
Fixing chucklefish logo size (Issue #655)

### PR DESCRIPTION
**Pull request by Anish-Agnihotri**
**Intro**

Hey folks! Noticed that #655 was significant and had some discussion about potential size reductions. I've made this quick PR that adds a one-liner html id, and a few lines of responsive CSS to make sure that the chucklefish logo is a better fit at all screen sizes.

Looking to help out with some of the open issues, and went through the Contributors guidelines, but do let me know if I'm out of place!

Thanks,

Anish

**New sizing screenshots**

The new sizing should look better on smaller displays and be less obtrusive on larger ones. It is responsively sized based on width: percentage.

Large display:
large_display

Medium display:
medium_size

Mobile display: 
mobile_reduction

**Resolves**
#655

### CODE REVIEW

**Comment by Anish-Agnihotri** [comment]
On the same note, would it be a better idea to keep things simple and simply resize the source PNG?

File: templates/components/panels/production.hbs [hunk 1]
```diff
@@ -22,7 +22,7 @@
           <p class="attribution">&ndash; Catherine West, Technical Lead</p>
         </div>
         <div class="four columns">
-          <a href="https://blog.chucklefish.org/about/" target="_blank" rel="noopener"><img src="/static/images/user-logos/chucklefish.png" alt="Chucklefish Logo" /></a>
+          <a href="https://blog.chucklefish.org/about/" target="_blank" rel="noopener"><img src="/static/images/user-logos/chucklefish.png" alt="Chucklefish Logo" id="chucklefish" /></a>
```
Comment (yoshuawuyts) [inline templates/components/panels/production.hbs]: We could perhaps express this with less code using Tachyons:
```suggestion
          <a href="https://blog.chucklefish.org/about/" target="_blank" rel="noopener"><img src="/static/images/user-logos/chucklefish.png" alt="Chucklefish Logo" class="w-33 w-50-ns" /></a>
```

Comment (Anish-Agnihotri) [reply templates/components/panels/production.hbs]: Make complete sense. My apologies, I'll pay more attention to simple tachyon-backed solutions.

Just pushed another commit that removes the class based width style: `width: 80%` and instead gives the images a tachyons `w-` attribute.

**Comment by yoshuawuyts** [comment]
> On the same note, would it be a better idea to keep things simple and simply resize the source PNG?

I think the approach you've taken is really good, and the results look great! :tada:

**Pull request merged**